docs/docs/preview/iframebridge/IFrameBridge.md

```diff
@@ -0,0 +1,65 @@
+---
+title: IFrameBridge
+sidebar_position: 8
+---
+
+The Block Preview allows you to preview your blocks in the context of the Comet Admin. The Block Preview is an iframe that is embedded in the Comet Admin. Comet Admin is intended to be a headless CMS, but to offer the capability to render a preview of the content, the block-preview got introduced.
+
+![IFrame](images/iframe.png)
+
+The `IFrameBridge` is a communication bridge between the Comet Admin and the Block Preview implemented in the site's client. It allows you to send messages from the Comet Admin to the Block Preview and vice versa. In the admin client the `IFrameBridge` is already implemented and can be used out of the box and is already sending messages to the iframe site client's block preview. In the site client you can use the IFrameBridge from `@comet/cms-site` to receive/send messages from/to the Comet Admin.
```

Comment:
Regarding "site's client": Isn't site and client the same?

Reply:
agree, i though it would be a little bit clearer when haveing both words, but we can reduce it to client.